jupyter/datascience/ubi9-python-3.11/Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY Pipfile.lock ./
 
 # TO DO: Uncomment this lines when elyra compatibility with jupyterlab 4.2 is done
```

Comment:
Could you remove as well the comment `# TO DO: Uncomment this lines when elyra compatibility with jupyterlab 4.2 is done`